pkg/compose/pull.go

```diff
@@ -133,6 +133,19 @@ func (s *composeService) pull(ctx context.Context, project *types.Project, opts
 	return err
 }
 
+func imageAlreadyPresent(serviceImage string, localImages map[string]string) bool {
+	normalizedImage, err := reference.ParseDockerRef(serviceImage)
+	if err != nil {
+		return false
+	}
+	tagged, ok := normalizedImage.(reference.NamedTagged)
+	if !ok {
+		return false
+	}
+	_, ok = localImages[serviceImage]
+	return ok && tagged.Tag() != "latest"
```

Comment:
Does `ParseDockerRef` normalize a missing tag to `latest`?

Reply:
yup!